yangify/linter/__init__.py

```diff
@@ -26,6 +26,7 @@
 
 class MessageType:
     PATH_EMPTY = "E001"
+    PATH_MISSING_SLASH = "E002"
```

Comment:
Should this be a warning instead of an error?  If the path doesn't begin with a `/`, everything still works fine.  I think it's more cosmetic than functional, hence the lint.

Reply:
Totally missed that lol Fixed